tensorboard/plugins/debugger/README.md

```diff
@@ -0,0 +1,144 @@
+# The Debugger Dashboard
+
+The debugger dashboard offers a graphical user interface for the [TensorFlow debugger](https://www.tensorflow.org/versions/r1.1/programmers_guide/debugger). For instance, this dashboard
+
+* enables users to pause execution at specified ops or continue execution for some number of steps.
+* visualizes values of tensors.
+* associates tensors with specific lines in python code.
+
+![The Debugger Dashboard](images/debugger_intro.png)
```

Comment:
Can you add and INFO section stressing that we are in the `alpha` release, and not all features may be fully functional yet.